components/log-viewer-webui/client/src/pages/SearchPage/SearchControls/TimeRangeInput/index.module.css

```
@@ -0,0 +1,15 @@
+.timeRangeInputContainer {
+    display: inline-flex;
+}
+
+/* Makes border flush with range picker */
+.select-container :global(.ant-select-selector) {
+    border-top-right-radius: 0 !important;
+    border-bottom-right-radius: 0 !important;
+}
+
+.rangePicker {
+    border-top-left-radius: 0 !important;
+    border-bottom-left-radius: 0 !important;
```

it doesn't seem `!important` is required for specifying the styles here:
```suggestion
    border-top-left-radius: 0;
    border-bottom-left-radius: 0;
```